pkg/services/network/constants.go

```diff
@@ -19,7 +19,7 @@ package network
 
 import (
 	netopv1 "github.com/vmware-tanzu/net-operator-api/api/v1alpha1"
-	nsxopv1 "github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
+	nsxvpcv1 "github.com/vmware-tanzu/nsx-operator/pkg/apis/vpc/v1alpha1"
```

Comment:
let's add a linter rule for this import in `.golangci.yml`

Reply:
Done. Please take a look if I set it correctly.

Reply:
Looks perfect, thx!